samples/system-test/profanityFilter.test.js

```diff
@@ -30,3 +30,4 @@ describe('profanityFilter', () => {
     assert.match(stdout, /Transcription:/ );
   });
 });
+
```

Comment:
nit: looks like a newline added that wasn't needed (again the linter letting you down).